test/test_iMPOConversions.jl

```diff
@@ -1,6 +1,35 @@
 using ITensors
 using ITensorInfiniteMPS
 using Test
+
+#With the new definition of InfiniteMPOMatrix, the MPO is better behaved, and hence we need to be a bit more careful
+function special_expect(ψ::InfiniteCanonicalMPS, h::InfiniteSum{MPO})
```

Comment:
Do we need both `expect` and `special_expect`? It sounds like the reason for defining `special_expect` is that `InfiniteMPOMatrix` changed a bit, so then should we just remove the old version of `expect` and rename this to `expect`?

Reply:
I realized after I commented that this is just used in tests so the name doesn't matter so much, but I would use `_expect` instead of `special_expect` since "special" isn't very descriptive.